waterlog/src/Pages/Home.js

```diff
@@ -10,7 +10,7 @@ class HomeComponent extends Component {
         <AlertsMap />
         <Wastage />
         <p className="home-text">Tank Levels</p>
-        <TankLevels />
+        {/*<TankLevels />*/}
```

Comment:
Why comment out this component?

Reply:
The component has errors, the person responsible for it is fixing it